### PR TITLE
fix: adds accessibile labels to moderation buttons

### PR DESCRIPTION
**Pull request by thisislawatts**
PR Checklist

- [x] - Commit [messages are descriptive](https://github.com/ONEARMY/community-platform/blob/master/CONTRIBUTING.md#--commit-style-guide), it will be used in our [Release Notes](https://github.com/ONEARMY/community-platform/releases/)

PR Type

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Developer experience (improves developer workflows for contributing to the project)

## Description

Use new `showIconOnly` property and use margin-left
for consistent layout.

## Screenshots/Videos

**Before:**

Screenshot 2023-01-08 at 14 18 52

**After:** 

Screenshot 2023-01-08 at 14 18 35

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -215,7 +215,8 @@ jobs:
             rm package.json
             npm config set package-lock false
             npm i @types/node @commitlint/types @commitlint/config-conventional --force
-            npx commitlint --from=$(git merge-base remotes/origin/${CIRCLE_BRANCH} master) --verbose
+            npm ls
```
Comment (chrismclarke) [inline .circleci/config.yml:218]: nit(non-blocking)
Do we still need this line?

**Review by chrismclarke — APPROVED** [review]
One minor comment from me but otherwise happy with the code. I'll leave it to @davehakkens if there's any related design issues

**Review by davehakkens — APPROVED** [review]

**Review by chrismclarke — CHANGES_REQUESTED** [review]
@thisislawatts 
I tried to manually remove the additional `npm-ls` call however when running locally it seems like the commitlint config is not working
```
PS C:\apps\oneArmy\community-platform> git commit -m "chore: remove ci npm ls"
⧗   input: chore: remove ci npm ls
✖   Please add rules to your `commitlint.config.js`
    - Getting started guide: https://git.io/fhHij
    - Example config: https://git.io/fhHip [empty-rules]

✖   found 1 problems, 0 warnings
ⓘ   Get help: https://github.com/conventional-changelog/commitlint/#what-is-commitlint
```

Would you be able to either try fixing (I'm guessing something to do with the version installed or difference with commitlint vs @commitlint/cli), or moving the config changes into a separate PR so that we can still merge the labels update?

**Comment by chrismclarke** [comment]
On second thoughts I could probably try be a bit more useful (!)
I've moved the commitlint changes to #2070 and will merge the button changes here once tests pass

**Pull request closed without merging**